docs/developers/bitesize/workflowExecution/majorActors.md

```diff
@@ -0,0 +1,51 @@
+# Workflow Execution: Major Actors
+
+* **Bitesize Documentation:** Covering the main actors involved in workflow execution.
+* **Word Count:** 213
+
+## High Level Overview
+
+![high level overview diagram](WorkflowExecutionHighLevelOverview.png)
+
+## Actors and their Purposes
+
+### WorkflowManagerActor
+
+The `WorkflowManagerActor` is responsible for:
+
+* Polling the `WorkflowStore` at pre-configured intervals.
+* Starting new workflows
+* Tracking, supervising and aborting running workflows
+* Parent actor for all `WorkflowActor`s
+
+### WorkflowActor(s)
+
+Each `WorkflowActor` is responsible for:
+
+* Running a single workflow.
+* Starting jobs and sub-workflows as soon as they are able to run.
+  * Based on values in the (in-memory) ValueStore and ExecutionStore objects.
+* Parent actor for all `EngineJobExecutionActor`s and `SubWorkflowExecutionActor`s.
+
+### EngineJobExecutionActor(s)
+
+Each `EngineJobExecutionActor` (EJEA) is responsible for:
+
+* Running a single job.
+  * A "job" is a command line instruction to run on a backend.
```

Comment:
Note to self: these sub-bullets don't seem to render at the right indent level in readthedocs: https://cromwell.readthedocs.io/en/cjl_bitesize_workflow_execution_actors/developers/bitesize/workflowExecution/majorActors/